tests/e2e/s/simplex.go

```diff
@@ -0,0 +1,199 @@
+				Genesis:      genesisBytes,
+				PreFundedKey: key,
+				VersionArgs:  []string{"version-json"},
+			},
+		},
+		ValidatorIDs: tmpnet.NodesToIDs(nodes...),
+	}
+}
+
+func SimplexSubnetsOrPanic(nodes ...*tmpnet.Node) []*tmpnet.Subnet {
+	key, err := secp256k1.NewPrivateKey()
+	if err != nil {
+		panic(err)
+	}
+
+	return []*tmpnet.Subnet{
+		NewSimplexSubnetOrPanic(simplexSubnetName, key, nodes...),
+	}
+}
+
+var _ = e2e.DescribeSimplex("Create a Simplex [L1]", func() {
+	tc := e2e.NewTestContext()
+	require := require.New(tc)
+
+	ginkgo.It("Issue and Finalize a Transaction", func() {
+		network := e2e.GetEnv(tc).GetNetwork()
+
+		simplexSubnet := network.GetSubnet(simplexSubnetName)
+		require.NotNil(simplexSubnet)
+
+		simplexChain := simplexSubnet.Chains[0]
+		simplexValidators := getNodesForIDs(network.Nodes, simplexSubnet.ValidatorIDs)
+		require.NotEmpty(simplexValidators)
+
+		sortNodes(simplexValidators)
+
+		// advance two rounds
+		buildBlock(tc, simplexChain, simplexValidators)
+		buildBlock(tc, simplexChain, simplexValidators)
+
+		_ = e2e.CheckBootstrapIsPossible(tc, network)
+	})
+})
+
+// Builds and proposes a new block by issuing a transfer transaction from the leader of the next round.
+// Verifies that all nodes have accepted the block and advanced to the next round.
+func buildBlock(tc *e2e.GinkgoTestContext, chain *tmpnet.Chain, nodes []*tmpnet.Node) {
+	require := require.New(tc)
+
+	// grab the current round
+	client := api.NewClient(nodes[0].GetAccessibleURI(), chain.ChainID.String())
+	_, latestBlock, err := client.LastAccepted(tc.DefaultContext())
+	require.NoError(err)
+
+	round := latestBlock.Height + 1
+	leader := getLeaderForRound(nodes, round)
+	tc.Log().Info("current height and round",
+		zap.Uint64("height", latestBlock.Height),
+		zap.Uint64("round", round),
+		zap.Stringer("leader", leader.NodeID),
+	)
+
+	// record the nonces of all nodes before issuing the tx
+	nonces := make(map[ids.NodeID]uint64)
+	for _, node := range nodes {
+		client = api.NewClient(node.GetAccessibleURI(), chain.ChainID.String())
+		nonce, err := client.Nonce(tc.DefaultContext(), chain.PreFundedKey.Address())
+		require.NoError(err)
+		nonces[node.NodeID] = nonce
+	}
+
+	// issue a transfer transaction to the leader
+	leaderURI := leader.GetAccessibleURI()
+	tc.Log().Info("issuing XSVM transfer transaction for simplex subnet",
+		zap.Stringer("leader", leader.NodeID),
+		zap.String("leaderURI", leaderURI),
+	)
+	recipientKey := e2e.NewPrivateKey(tc)
+	transferTxStatus, err := transfer.Transfer(
+		tc.DefaultContext(),
+		&transfer.Config{
+			URI:               leaderURI,
+			ChainID:           chain.ChainID,
+			AssetID:           chain.ChainID,
+			Amount:            units.Schmeckle,
+			To:                recipientKey.Address(),
+			PrivateKey:        chain.PreFundedKey,
+			WaitForAcceptance: false,
+		},
+	)
+	require.NoError(err)
+
+	// issue txs to all other nodes to ensure they are aware of the tx
```

Comment:
why do we do that if the leader will broadcast a block to the rest?

I am confused. 

`transfer.Transfer` issues a transaction to the leader, and then we do it again but for all nodes. 
But if we do it again then why do we care who is the leader? If we just issue it to everyone.